app/lib/data.ts

```diff
@@ -77,26 +77,21 @@ export function getLedgerDisplayName(ledger: string | undefined): string {
   // Return a clean, readable version
   switch (normalized) {
     case "candy-prod":
-      return "Candy Production";
+      return "CANdy Production";
```

Comment:
Now that we have a convention for did:webvh, you can include it here.

- webvh:bcgov:sandbox --> BC Gov did:webvh Sandbox Server
- webvh:bcgov:dev --> BC Gov did:webvh Development Server
- webvh:bcgov:test --> BC Gov did:webvh Test Server
- webvh:bcgov:prod --> BC Gov did:webvh Production Server